py4syn/epics/DxpClass.py

```diff
@@ -104,25 +113,42 @@ def getValueChannel(self, **kwargs):
         channel = kwargs['channel']
         c = int(channel[3]) - 1
         if(len(channel) > 4):
-            r = int(channel[5])
-            return self.pvDxpRois[c][r]
+            r = int(channel[6])
```

Comment:
How is this ```getValueChannel``` being used?
Why do we need to have this index 6 hardcoded? 
If it is necessary, I would recommend the use of a constant in order to make it easier to understand now and in the future.